server.js

```diff
@@ -1,10 +1,12 @@
 const express = require('express');
 const app = express();
+const AccountOperationsManager = require('./controllers/AccountOperationsManager'); 
+const bodyParser = require('bodyParser'); 
```

Comment:
We don't need that anymore :) 
```suggestion
app.use(express.json())
```